<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetrePrincipale</class>
 <widget class="QMainWindow" name="FenetrePrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>522</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="onglet_analyse">
       <attribute name="title">
        <string>Analyse d'une liste de mots</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <widget class="QPushButton" name="bouton_analyser">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Lancer l'analyse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_choixListe">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0" colspan="2">
            <widget class="QRadioButton" name="radio_langDef">
             <property name="text">
              <string>Utiliser la langue par défaut (////Nom liste défaut////)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radio_langPerso">
             <property name="text">
              <string>Choisir une liste de mot</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_choixLang">
             <property name="text">
              <string>Choix de la liste de mots à analyser :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="bouton_selecFichier">
             <property name="text">
              <string>(Aucun)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame_mode">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="1" colspan="2">
            <widget class="QRadioButton" name="radio_speciaux">
             <property name="toolTip">
              <string>C'est beau l'utf-8, mais c'est chiant à utiliser !</string>
             </property>
             <property name="text">
              <string>Gère tout les caractères spéciaux</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="4" rowspan="4">
            <widget class="Line" name="line_details">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_lcoh">
             <property name="text">
              <string>Longueur de cohérence :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spin_lcoh">
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string> lettres</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="Qualite">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Qualité</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QLabel" name="Bof">
             <property name="toolTip">
              <string>Pour ce que le programme fait (lire des suites de lettres. Noter le nombre d'occurence),
cette méthode est honteusement longue</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(223, 177, 19);</string>
             </property>
             <property name="text">
              <string>Bof</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="Parfait_2">
             <property name="toolTip">
              <string>C'est presque instantanné. Normal, y a vraiment pas besoin d'un supercalculateur pour faire ça</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(44, 161, 40);</string>
             </property>
             <property name="text">
              <string>Parfait</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="Bof_2">
             <property name="toolTip">
              <string>On génère pas des mots accentués. Et en plus, ça gère pas les tirets</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(223, 177, 19);</string>
             </property>
             <property name="text">
              <string>Bof</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="Minable">
             <property name="toolTip">
              <string>J'voulais pas faire un truc pour les anglais, moi !</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>Minable</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="Rapidite">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Rapidité</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="Parfait">
             <property name="toolTip">
              <string>Testez avec le Japonnais ! Promis, ça marche !</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(44, 161, 40);</string>
             </property>
             <property name="text">
              <string>Parfait</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLabel" name="Minable_2">
             <property name="toolTip">
              <string>En même temps, c'est codé avec les pieds</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>Minable</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QRadioButton" name="radio_ignore">
             <property name="toolTip">
              <string>Les lettres accentuées sont supprimée par l'analyse : on regarde le mot &quot;ttt&quot; au lieu de &quot;étêtât&quot;
Il vaut donc mieux que la liste de mots originale soit sans accents</string>
             </property>
             <property name="text">
              <string>Ignore les accents</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QRadioButton" name="radio_minable">
             <property name="toolTip">
              <string>Ces caractères spéciaux sont remplacé par leur valeur non-accentué :
ÀÁÂÃÄÅàáâãäåÒÓÔÕÖØòóôõöøÈÉÊËèéêëÌÍÎÏìíîïÙÚÛÜùúûüÿÑñÇç</string>
             </property>
             <property name="text">
              <string>Gère très mal les accents</string>
             </property>
            </widget>
           </item>
           <item row="4" column="5">
            <widget class="QPushButton" name="bouton_details">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>(+ détails)</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_mode">
             <property name="text">
              <string>Mode de traitement :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2" colspan="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QProgressBar" name="progr_Analyse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>15</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="onglet_generation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Génération de mots</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="5" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="bouton_trier">
          <property name="text">
           <string>Trier</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="bouton_nettoyer">
          <property name="text">
           <string>Nettoyer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_tailleMots">
          <property name="text">
           <string>Taille max des mots</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QPushButton" name="bouton_generer">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Inventer des mots</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="check_forcerTaille">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Forcer la taille des mots</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spin_tailleMax">
          <property name="suffix">
           <string> lettres</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="9">
         <widget class="QTextEdit" name="text_mots"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spin_nbMots">
          <property name="cursor">
           <cursorShape>SizeVerCursor</cursorShape>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="check_troll">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Génération de mots parfaits</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_nbMots">
          <property name="text">
           <string>Nombre de mots</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" rowspan="2" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_resume_analyse">
             <property name="text">
              <string>Type d'analyse : //type//</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_resume_lcoh">
             <property name="text">
              <string>Longueur de cohérence : //lcoh//</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuQuitter">
     <property name="title">
      <string>Quitter</string>
     </property>
     <widget class="QMenu" name="menuQuitter_2">
      <property name="title">
       <string>Quitter</string>
      </property>
      <addaction name="actionNe_pas_quitter"/>
      <addaction name="actionNe_pas_quitter_2"/>
      <addaction name="actionNe_pas_quitter_3"/>
      <addaction name="actionQuitter_3"/>
      <addaction name="actionNe_pas_quitter_4"/>
     </widget>
     <addaction name="menuQuitter_2"/>
    </widget>
    <addaction name="actionCharger_automatiquement_la_liste_de_mot_par_d_faut"/>
    <addaction name="actionAnalyser_la_liste_par_d_faut_au_lancement"/>
    <addaction name="separator"/>
    <addaction name="actionUtiliser_les_valeur_actuelles_par_defaut"/>
    <addaction name="separator"/>
    <addaction name="menuQuitter"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="action_propos_de_ce_programme"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter_3">
   <property name="text">
    <string>Ha oui, quitter</string>
   </property>
  </action>
  <action name="actionNe_pas_quitter">
   <property name="text">
    <string>Ne pas quitter</string>
   </property>
  </action>
  <action name="actionNe_pas_quitter_2">
   <property name="text">
    <string>Ne pas quitter</string>
   </property>
  </action>
  <action name="actionNe_pas_quitter_3">
   <property name="text">
    <string>Ne pas quitter</string>
   </property>
  </action>
  <action name="actionNe_pas_quitter_4">
   <property name="text">
    <string>Ne pas quitter</string>
   </property>
  </action>
  <action name="actionUtiliser_les_valeur_actuelles_par_defaut">
   <property name="text">
    <string>Changer les valeurs par défaut</string>
   </property>
  </action>
  <action name="actionCharger_automatiquement_la_liste_de_mot_par_d_faut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Charger la liste par défaut au lancement</string>
   </property>
  </action>
  <action name="actionAnalyser_la_liste_par_d_faut_au_lancement">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Analyser la liste par défaut au lancement</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="action_propos_de_ce_programme">
   <property name="text">
    <string>À propos de ce programme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radio_speciaux</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_lcoh</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
